static/src/javascripts/projects/journalism/snippet-feedback.js

```diff
@@ -0,0 +1,84 @@
+// @flow
+import fastdom from 'lib/fastdom-promise';
+import mediator from 'lib/mediator';
+import ophan from 'ophan/ng';
+import detect from 'lib/detect';
+
+const SnippetFeedback = () => {
+    const snippets = document.querySelectorAll('.explainer-snippet--new');
```

Comment:
You could convert the `NodeList` right here into an array:

```js
  const snippets = [...document.querySelectorAll('.explainer-snippet--new')]; 

  snippets.forEach(snippet => { /*...*/ });
```

Bit more readable. Who do you think?